### PR TITLE
feat: add a create/delete guild event

### PR DESCRIPTION
**Pull request by a2937**
# Pull Request

## Description:

It adds a create and delete guild event to track which servers the bot is on. 

## Related Issue:

Closes #XXXXX

### CODE REVIEW

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: src/events/_handleEvents.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { Guild } from "discord.js";
```
[the remark this answers is not in the document]
Comment (a2937) [reply src/events/_handleEvents.ts]: Removed unnecessary import.

File: src/events/guild/onGuildCreate.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      guild.channels.cache.get(config.eventLogChannel) ||
+      (await guild.channels.fetch(config.eventLogChannel));
+
+    if (!channel || !("send" in channel)) {
+      return;
+    }
+
+    await channel.send({
+      content: `The bot has joined ${guild.name}${guild.id} owned by ${user.tag} (${user.id})`
+    });
```
[the remark this answers is not in the document]
Comment (a2937) [reply src/events/guild/onGuildCreate.ts]: Oops. I've fixed it now.

File: src/events/guild/onGuildDelete.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      guild.channels.cache.get(config.eventLogChannel) ||
+      (await guild.channels.fetch(config.eventLogChannel));
+
+    if (!channel || !("send" in channel)) {
+      return;
+    }
+
+    await channel.send({
+      content: `The bot has left ${guild.name}${guild.id} owned by ${user.tag} (${user.id})`
+    });
```
[the remark this answers is not in the document]
Comment (a2937) [reply src/events/guild/onGuildDelete.ts]: Got it.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
## Thank You
  
  Thank you for your contribution to our project. We have reviewed your pull request and are happy to accept these changes.
  
  Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.
  
  You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt). Also, feel free to join our [Discord server](https://chat.naomi.lgbt) to chat with us and get notified when new issues are available!

**Pull request merged**